.eslintrc.cjs

```diff
@@ -12,7 +12,7 @@ module.exports = {
 	overrides: [
 		{
 			extends: ["plugin:markdown/recommended"],
-			files: ["**/*.{md}"],
+			files: ["**/*.md"],
```

Comment:
Right, I'd removed this from the other PR because it was causing issues - then forgot to file a followup issue to look into it. 😅 

I need to figure out how to reconcile embedded TS snippets in Markdown docs...

```plaintext
/home/runner/work/template-typescript-node-package/template-typescript-node-package/README.md
  96:1  error  Parsing error: ESLint was configured to run on `<tsconfigRootDir>/README.md/2_2.ts` using `parserOptions.project`: <tsconfigRootDir>/tsconfig.eslint.json
```

Reply:
Right, didn't catch that, sorry. Wow TIL `files: ["**/*.md/*.js"]` to lint `js` fenced code blocks, that's pretty cool :)

Reply:
#208, fun! 🙃

Reply:
Oh my, it's literally creating those files. But adding them to `includes` doesn't seem to help. That's a nasty conflict of plugins then, it seems.

But then the markdown plugin didn't actually do something here in this repo, then?

Reply:
I'm ... guessing not 😂